app/src/geckoBeta/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -33,6 +34,7 @@ object GeckoProvider {
         val runtimeSettings = builder
             .crashHandler(CrashHandlerService::class.java)
             .useContentProcessHint(true)
+            .telemetryDelegate(GeckoAdapter())
```

Comment:
Does this require GV 70? If so then this requires landing PR #5114 first. Until then the geckoBeta flavor is still on 69.

Reply:
Yes, it requires 70. I wonder why this doesn't fail locally nor on CI. Oh well :o

Reply:
CI unfortunately doesn't build all flavors currently.